src/store/reducers/deploymentsReducer.js

```diff
@@ -161,6 +161,14 @@ const updateDeployedDevicesReducer = (state, { payload: [modules, devices], from
   });
 };
 
+const resetDeployedDevicesReducer = (state) => {
+  return update(state, {
```

Comment:
nit: Use the implicit return where possible
```javascript 
const resetDeployedDevicesReducer = (state) => update(state, {
  ..// stuff
});
```

Reply:
done